index.html

```diff
@@ -22,12 +22,18 @@ <h1 class="title-header">Welcome to Acme Web Sulutions</h1>
           consectetur, adipisci velit...
         </div>
         <div class="button-container">
-          <button class="button-header">
+          <button
+            aria-label="jump to seconed section"
+            tabindex="0"
+            role="button"
```

Comment:
is it necessary to put role = button here? doesn't the button tag have it by default? or am i missing sth?

Reply:
Oh!You are right.